department-of-veterans-affairs/va.gov-cms
# VACMS-4832: Make va_gov_migrate D9-compatible.

## Description

See #4832.  A drush command wasn't picked up previously, presumably because we hadn't upgraded to Drush 10 at that point. 

## Testing done
Ran the command and it executed successfully.  Re-ran Upgrade Status and no issues were found.

## Screenshots
<img width="897" alt="Screen Shot 2021-03-29 at 9 57 47 AM" src="https://user-images.githubusercontent.com/1318579/112847752-64b3a400-9075-11eb-93ef-b734942eb30e.png">
<img width="670" alt="Screen Shot 2021-03-29 at 9 55 43 AM" src="https://user-images.githubusercontent.com/1318579/112847762-6715fe00-9075-11eb-88d5-25e8791650e3.png">

## QA steps

As a developer:
1. Run the command and verify that it doesn't fail from code modifications (I relocated the code and injected dependencies).
1. Then validate Acceptance Criteria from issue #4832:
- [ ] `va_gov_migrate` should no longer present any issues with an upgrade to Drupal 9 in the Upgrade Status report.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
